google/cloud/bigquery/client.py

```diff
@@ -2162,7 +2163,9 @@ def get_job(
                 object.
             retry (Optional[google.api_core.retry.Retry]):
                 How to retry the RPC.
-            timeout (Optional[float]):
+            timeout (Optinal[Union[float, \
+                google.api_core.future.polling.PollingFuture._DEFAULT_VALUE, \
```

Comment:
Same here. `Client` shouldn't have to know about the `PollingFuture` object.

Reply:
done

Reply:
Reverted too.